MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -585,6 +585,31 @@ public bool IsTextSubtitleStream
             }
         }
 
+        public bool IsPgsSubtitleStream
+        {
+            get
+            {
+                if (Type != MediaStreamType.Subtitle)
+                {
+                    return false;
+                }
+
+                if (string.IsNullOrEmpty(Codec) && !IsExternal)
+                {
+                    return false;
+                }
+
+                return IsPgsFormat(Codec);
+            }
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether this is a subtitle steam that is extractable by ffmpeg.
+        /// All text-based pgs subtitles can be extracted.
```

Comment:
text-based pgs? (Presentation Graphic Stream)

Reply:
Right ... I missed an "and" there!